src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import { getUser, getTodos } from '../../api';
+
+type Props = {
+  selectUser: number
+  selectTodo: number,
+  selectItems: (UserId: number, todoId: number) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectUser,
+  selectTodo,
+  selectItems,
+}) => {
+  const [user, setUser] = useState<undefined | User>();
+  const [todo, setTodo] = useState<undefined | Todo>();
```

Comment:
```suggestion
  const [user, setUser] = useState<User | null>(null);
  const [todo, setTodo] = useState<Todo | null>(null);
```
better to set null as the default value